src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/modalwindows/CuratedContentToolModal.java

```diff
@@ -0,0 +1,30 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.modalwindows;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * Created by wikia on 2015-08-19.
+ */
+public class curatedContentToolModal extends WikiBasePageObject {
+
+  @FindBy(css = "#CuratedContentToolModal")
+  protected WebElement CuratedContentToolModal;
+
+  public curatedContentToolModal(WebDriver driver) {
+    super(driver);
+  }
+
+  public boolean isModalVisible() {
+    try {
+      WebElement modal = wait.forElementVisible(CuratedContentToolModal);
+      return modal.isDisplayed();
+    } catch (TimeoutException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)